src/internal/observable/fromEvent.ts

```diff
@@ -177,18 +189,18 @@ export function fromEvent<T>(
   });
 }
 
-function setupSubscription<T>(sourceObj: EventTargetLike, eventName: string,
-                              handler: Function, subscriber: Subscriber<T>,
+function setupSubscription<T>(sourceObj: FromEventTarget<T>, eventName: string,
+                              handler: (...args: any[]) => void, subscriber: Subscriber<T>,
                               options?: EventListenerOptions) {
   let unsubscribe: () => void;
-  if (isNodeList(sourceObj) || isHTMLCollection(sourceObj)) {
-    for (let i = 0, len = sourceObj.length; i < len; i++) {
+  if (sourceObj && (sourceObj as any).length) {
```

Comment:
You can use `isArrayLike` here, right? https://github.com/ReactiveX/rxjs/blob/master/src/internal/util/isArrayLike.ts